### DISCUSSION TITLE
import error when import GCNConv

### DISCUSSION THREAD

**Discussion by Catriminal** [pyg-team/pytorch_geometric#2785] [Q&A]
Hi, after I install torch_geometric and I try to import GCNConv, I got this:

```
*** Error in `python': corrupted size vs. prev_size: 0x0000559df70d6bd0 ***
```

and sometimes I can't get any response when I run

```python
import torch
from torch_geometric.nn import GCNConv
```

My python version is 3.6 and cuda version is 10.1.

Here is how I install torch and torch_geometric. Hope it can help.

```bash
pip install torch==1.6.0+cu101 torchvision==0.7.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html

pip install torch-scatter -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-sparse -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-cluster -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-spline-conv -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-geometric
```

Here is the versions of what I installed.

```
torch                 1.6.0+cu101
torch-cluster         1.5.9
torch-geometric       1.7.2
torch-scatter         2.0.7
torch-sparse          0.6.10
torch-spline-conv     1.2.1
torchvision           0.7.0+cu101
```

Is there any advice about this?  Thanks for your reading.

**Comment by rusty1s** [comment]
You should define the version numbers of external packages in case you are using an older PyTorch version as we don't build new versions of our packages for older PyTorch versions. Those can be found [here](https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html):
```
pip install torch-scatter==2.0.6 -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-sparse==0.6.9 -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-cluster==1.5.9 -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-spline-conv==1.2.1 -f https://pytorch-geometric.com/whl/torch-1.6.0+cu101.html
pip install torch-geometric
```

**Reply by Catriminal** [reply]
Thanks for your reply! I installed the versions of packages you mentioned, but it still can't work. I got `Segmentation fault (core dumped)` when I try to import torch_geometric.

Here is the versions of packages:

```
torch                 1.6.0+cu101
torch-cluster         1.5.9
torch-geometric       1.7.2
torch-scatter         2.0.6
torch-sparse          0.6.9
torch-spline-conv     1.2.1
torchvision           0.7.0+cu101
```

Besides, I created a new environment with python 3.7. And installed the same version of packages, it worked. So I guess there may be some problems in my previous environment.
